modules/fscloud/version.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.3.0"
+  required_providers {
+    # The below tflint-ignore is required because although the below provider is not directly required by this submodule,
+    # it is required by consuming modules, and if not set here, the top level module calling this module will not be
+    # able to set alternative alias for the provider.
+    # tflint-ignore: terraform_unused_required_providers
+    ibm = {
+      source = "IBM-Cloud/ibm"
+      # Use "greater than or equal to" range in modules
+      version = ">= 1.54.0"
```

Comment:
this now need to be updated to >= 1.56.1 since https://github.com/terraform-ibm-modules/terraform-ibm-icd-rabbitmq/pull/22 was merged